src/components/layout/navbar.tsx

```diff
@@ -45,6 +61,18 @@ const NavBar = () => {
     auth?.logout();
   };
 
+  /**
+   * fetch avatars
+   */
+  const getSlackAvatars = async () => {
+    const fetchedAvatars = await slackAvatarsApi.slackAvatar();
```

Comment:
This should be in a try/ catch as currently when the request fails (there is already another task to investigate/ fix this) it gives an uncaught error